frontend/src/container/ListAlertRules/ListAlert.tsx

```diff
@@ -67,6 +72,7 @@ function ListAlert({ allAlertRules, refetch }: ListAlertProps): JSX.Element {
 		{
 			title: 'Alert Name',
 			dataIndex: 'alert',
+			width: 100,
```

Comment:
do need to explicitly add the width in the initial rendering ?

Reply:
@palashgdev 
This is for resize header after initial rendering width. This is default width of header later on we can resize according to our conviction.